src/main/java/seedu/address/model/Model.java

```diff
@@ -4,9 +4,13 @@
 import java.util.function.Predicate;
 
 import javafx.collections.ObservableList;
+
```

Comment:
delete newline?

Reply:
I think this newline is needed, otherwise it would've been caught by checkstyle